client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -98,20 +130,47 @@ class TaxonomyManagerListItem extends Component {
 						<Gridicon icon="pencil" size={ 18 } />
 						{ translate( 'Edit' ) }
 					</PopoverMenuItem>
-					{ onDelete &&
-						<PopoverMenuItem onClick={ this.deleteItem }>
-							<Gridicon icon="trash" size={ 18 } />
-							{ translate( 'Delete' ) }
-						</PopoverMenuItem>
-					}
+					<PopoverMenuItem onClick={ this.deleteItem }>
+						<Gridicon icon="trash" size={ 18 } />
```

Comment:
I learned that `PopoverMenuItem` accepts an `icon` prop - so you can remove this line and add `icon="trash"` to the line above.